### PR TITLE
Sv lookup interval

### PR DESCRIPTION
**Pull request by hanars**
We were previously looking up matching SVs with a different sample type by just checking if there is any overlap between them. In fact, we should be checking that the breakpoints are within a specified tolerance of each other
Comparable ES functionality: https://github.com/broadinstitute/seqr/blob/master/seqr/utils/search/elasticsearch/es_search.py#L1427

### CODE REVIEW

File: hail_search/queries/sv.py
Comment (bpblanken) [inline hail_search/queries/sv.py:91]: Is [this](https://hail.is/docs/0.2/hail.expr.LocusExpression.html#hail.expr.LocusExpression.window) something that might be of use here?

Also, where does `3e8` come from?  Is it just "big number that's larger than the max pos on any config"?

**Review by bpblanken — APPROVED** [review]
don't wanna hold this up on style questions

File: hail_search/queries/sv.py
Comment (hanars) [reply hail_search/queries/sv.py:91]: its copied from xpos_utils in seqr, its what we use in ES to validate. `window` seems better but I will add it is a subsequent PR to unblock this

**Pull request merged**
